examples/sentiment/scripts/gpt-neox-20b_peft/README.md

```diff
@@ -0,0 +1,10 @@
+
+# Fine-tuning 20B LLMs with RL on a 24GB consumer GPU
+The scripts in the section detail the fine-tuning a 20b LLM in 8-bit, in order to generate positive imdb reviews. You
+can find out more in our [blogpost](LINK_TODO).
+
+Overall there were three key steps and training scripts:
+
+1. **s01_cm_finetune_peft_imdb.py** - Fine tuning a Low Rank Adapter on a frozen 8-bit model for text generation on the imdb dataset.
+2. **s02_merge_peft_adapter.py** - Merging of the adapter layers into the base model’s weights and storing these on the hub.
+3. **s03_gpt-neo-20b_sentiment_peft.py** - Sentiment fine-tuning of a Low Rank Adapter to create positive reviews.
```

Comment:
i'd be in favour of don't introduce a numbering scheme and just keep the semantic names. small nit `cm` -> `clm` or `lm`.

I think we could try to skip the first step for large models by engineering the prompt to be something like `"A positive movie review: {PROMPT}"`. I'd guess the model would already work quite well out of the box. Maybe for a future experiment.

Reply:
Good point.